new_template_bh6erk.html

```diff
@@ -0,0 +1,148 @@
EHYP2CrHGBDK9FmExV0NtBBDcveAQffxJfuAHnH3sFSa6fcjB2YChMClJpK7jw8eLjYLA1xILJWm8i+s/eGD3X4eR1nc0aDOWviyazFA62kku6QmXqjgHV+bnp6zTyTuBY92eYQHwKUmkltPA1oTJTknEiIlwqX+7px3BNGpQNQ0bW73NIRonUeAq9N3ZYDXtUnOiYRwJ5ZOB95Rg8FX5a+8L7RBCNGqNoEtpvHr2kBnF269vhbsxBmfb+7s9u3bZwcD9+vpSBcC1uk7cmsTIetLLdmEgXKTkneAcvCLe2DuFxdR2x91ol1AC1su5NeXQrsau15/0PePj+54PL/6p49/9Qx1xz9Iah+3AKxit2Vd5AJd5y351exeF/rHfQb/uO3pzLNTre8COpX6MKecR+rA1+rbckrb3LBf8sq3LTeDVLb+8gfi9IHdiUW3UL8M7Gha8OctLClltxLaPvm7GzmMPhf+Utu1z3NnUMUuBc5bAJ6ZseTV62vTB0jRg/gRwmltDsn9UAF48uSfmQE892Tr/jRFUcKQgF2pf+4J+iJB5e40+XYMMmHh+V3YLqxuwJ8/dKVI5EtqiIRAt+8EFDh/CHaGdCQiIokQ6O6doAJnSMHjhTXTJ9zEQyAEn91HzgFncCmrsolHxBOWQc8gOAqd5c7ATEn3KWEJYe5Oiih0Hj8DM7UmijhEHCHQdSZhRe5UyOTuYP/cM4OwADwrdBS5FyObWz0VTOzHEAKHQVeRu00yuj0YE/ujhOBx3lXsfhrw7QtH8dgfIYSP865idwxl966A2XJBC2g6+EUB6qbEmOL3RGV3gbCihCptgmvexTXY62cDwtz1lUVIdEQlzOqXYu5rg7lzDysKYVa/EnvnHvhGoq+bIMTdm5hJXmPr+gnxd1+C3SIc0/UTEu4vhbqDNiq9TibUs0lnCHfQZhH1q3p17bezKQJh/vFva4oOemjUFvEeYeBOVHWj/vvZ1K1wlVR4D7hYzD/+fc3QYX8x8S5oyJFYNar32g35cCqq+C73hdxYbqkGXGJMuc8b6k52pddcd16hNeQgHNqfvLPe7AEd5qLcyQ6ySaP06q9en3stchB6Hz1//aoOAUm9Vz/1uxGUXufV6xPTPPEajNUsxAhnvI/m0I+9ftVJC0l/N0LKmbDSe/MW4eWQvPYumIQXPiESgnz7Jh0j4/0WKaYYaPC9O3fw0JP6DUYv/YoS9v1Pej9qnr9LMSRjr5aFes+M0nuPui/nyRySOjFK+NL74HBp9NMnb98nZIy/zRLmXUHIPD+YI75cbt4nHM5QCUddOJwP/LxpfkhmrBzvCkqwYoP4rkJ8udyiTxjtxAihPwrlw8VQC8kYed73JP5eud7712E8qw8PR+2VKIQvRx/7bj7aiPn6fU/0YTA48X8SWztVOh9zMcDc/HeBBimEgU8dxQhRux87Qt1o8L13Tejdeb0353E+RHgUaG9IJBwGPnUcJ0SM528EupH33Xkidtr7iOPL5ZaOQw2WsIQzoc/8dwnbkvlRABELg/tH3ndYKp0PeMDc0vfhFi9mYoT9i/BHLglNma95LVXgHZacyZvSvCI8Vc68jLQ4LIUJ+y+HkU/8QWzrqsmFKPIeUs71YYUImDP/iDU5vJiacQn7+Ysonyx/Ijd2xWOoYu+S5XofcC8eJEYP9Rnb7PDC0iH2e+fExnLmWw5EVex9wBzvdFbekAFzuRNCu2ThHY2L+IZpp6LvdJblVVZU7L2lEc4LEy5SWjM/sDrR2CK1m/zd6grFrFDaJgp4SCPM5RhFi0nerS7Ldaq3Ud7TujC3eChIGE/aQp34jmqm8RkFF2GDTviOShhK23iESdqCortTkpdhENLXT2me1CI8pjSMEyGl8TvxPaUTQ+ujIoRUh6qcUJ8omtQwRUppPMI/yYREN8omlNtEROWfjCeKJjUsfaG3RzHTSnTdQoRQbpMMVfmTQYgP+WRhpyjBBkneNLYwI0YobxEQe6Sc29M5o+Go6K2RvSk5EHISkiI/YxgGV9u4NKS7UqR/Y820sspqmUkor+LGImsYhlZqeEQPh7ZwRwrYgByE8hYGkTUMhQMiIxwimf+Km2mFZaJ8hDiPyhyG4XUMtr6nh0OL8K8YIcOL8hPKyzHEDmsY5kyxgEic/44UixcVzLpTQsLYKUz2MMTNgWn6zCY0I4A6NdALEsoLaihJZSSltsTCBdNIEeHfQTNVVVqqJk4oN+rByRQjKbUlNH8a0udODuFfATOt1inJdiJCNF8MBMYedW7oEoqEi0N2sAgNRIM8H0xOGAiMyhq7C8XCBTtYWPIjIkcYTEIo17y6EOUVB+GSyPyJHSxyo4ioVvl8jDghGoyOT+39xUEoNLtgzSycFp0ZlM49BMUJZXnDttTeFc/ziMwuOMZ1zk1NDQELtSRIKN/pIJ/aZMZ7SyILijxGilrsSdUOdumeIlFCWW5V6CulvgScKWOhzZN5v4LbXaJLnFCusdNuWwLOlM+V5pb+FnExrhIQyvJ/FnkQBZwpY5HGkbkoujJiKxGhfHjO8Z8u4EzJuzIjzZ8fJnrWZISyfGyypztX3K1xpN2m6PKkp6SEyLLmmVNE3qaYSxjmfCIDtZWcUB5+YQzHYEEGVYwlDHPxk9iSSEgpCNFw/ERl5J7mU9e7Ed9hmodMRchg5J7mUyb4aflSEyLGL8TxaH7ibIOYs5nzXw7TPmBqQjQeL+dJkJwt4DMac37+MsX48wRAiHR8hTVWzrwN62jMxauk8SEsGEJkrJdL8Y7kdDVxR2POL10eAj0ZFCHS0ZeotXJmNRFHg6zzi9hiK1WAhLIFGepJziniSQhvCRJPhiZEOro8X1zyKLm2Z/yMxlxaPL+ExZMzIEQaHv9xggzW5MxqrKmTiUzz5MsxgOuMKQtCS8Ojy89Li4s87vD7xcWlz5dHWdBZyorQ1vCIx+aOMoOzlSnhSI2G88f/2v1zDfofvgkAFT5UgHAAAAAASUVORK5CYII=" />
```

Comment:
@bh6erk The **base64 encoded** image source doesn't seem appropriate. It **might be confusing for beginners** as this repo is supposed to be for beginners. Can you please provide a link to user-avatar as `src` of `<img />`?

_What do you think?_